core/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReport.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.rawls.model
+
+object ErrorReport extends ErrorReportBase {
+  def SOURCE = "rawls"
+}
```

Comment:
I think this is the thing should be called RawlsErrorReport, it is the part that is rawls specific.